integration_tests/e2e/induction/createCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -414,19 +414,20 @@ context(`Change links on the Check Your Answers page when creating an Induction`
       .removeQualification(1)
       .submitPage()
 
-    // Change reasons for not wanting to work on release
+    // Change affecting ability to work
     Page.verifyOnPage(CheckYourAnswersPage)
-      .clickReasonsForNotWantingToWorkChangeLink()
+      .clickFactorsAffectingAbilityToWorkChangeLink()
       .hasBackLinkTo(`/prisoners/${prisonNumber}/create-induction/check-your-answers`)
-      .deSelectReasonNotToGetWork(ReasonNotToGetWorkValue.FULL_TIME_CARER)
-      .chooseReasonNotToGetWork(ReasonNotToGetWorkValue.NO_RIGHT_TO_WORK)
-      .chooseReasonNotToGetWork(ReasonNotToGetWorkValue.RETIRED)
+      .deSelectAffectAbilityToWork(AbilityToWorkValue.CARING_RESPONSIBILITIES)
+      .chooseAffectAbilityToWork(AbilityToWorkValue.NO_RIGHT_TO_WORK)
+      .chooseAffectAbilityToWork(AbilityToWorkValue.RETIRED)
       .submitPage()
 
     // Then
     Page.verifyOnPage(CheckYourAnswersPage) //
       .hasHopingToWorkOnRelease(HopingToGetWorkValue.NO)
-      .hasReasonsForNotWantingToWork([ReasonNotToGetWorkValue.NO_RIGHT_TO_WORK, ReasonNotToGetWorkValue.RETIRED])
+      .hasFactorsAffectingAbilityToWork(AbilityToWorkValue.NO_RIGHT_TO_WORK)
+      .hasFactorsAffectingAbilityToWork(AbilityToWorkValue.RETIRED)
```

Comment:
minor thing but this seems like it would consider "all" rather than be "contains" based on the plural in the function name.  Maybe it should be an "all" though to ensure we don't end up with mystery items?

Reply:
Good shout; yes, I will do that 👍

Reply:
Done 👍